sys/net/gnrc/network_layer/ndp/gnrc_ndp.c

```diff
@@ -275,6 +275,70 @@ void gnrc_ndp_nbr_adv_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
     return;
 }
 
+#if (defined(MODULE_GNRC_NDP_ROUTER) || defined(MODULE_GNRC_SIXLOWPAN_ND_ROUTER))
+void gnrc_ndp_rtr_sol_handle(kernel_pid_t iface, gnrc_pktsnip_t *pkt,
+                             ipv6_hdr_t *ipv6, ndp_rtr_sol_t *rtr_sol,
+                             size_t icmpv6_size)
+{
+    gnrc_ipv6_netif_t *if_entry = gnrc_ipv6_netif_get(iface);
+    assert(if_entry != NULL);
```

Comment:
I think we can save the memory for the assert here, if the node should crash in the next line anyway if `if_entry` is NULL.